includes/definitions/tplink-eap.yaml

```diff
@@ -0,0 +1,12 @@
+os: tplink-eap
+text: 'TP-Link EAP'
+type: wireless
+icon: tplink
+over:
+    - { graph: device_bits, text: 'Device Traffic'}
+    - { graph: device_wireless_clients, text: 'Number of Clients' }      
+poller_modules:
+    wifi: true
+discovery:
+    - sysObjectID:
+            - .1.3.6.1.4.1.11863.3.2.10
```

Comment:
probably should just be .1.3.6.1.4.1.11863.3.

Reply:
Yes, definitely. My mistake